pipelines/internal/commands/run/run.go

```diff
@@ -200,7 +210,7 @@ func Invoke(ctx context.Context, service *genomics.Service, project string, argu
 
 		cancelOnInterruptOrTimeout(ctx, service, lro.Name, *timeout, abort)
 
-		fmt.Printf("Pipeline running as %q\n", lro.Name)
+		fmt.Printf("Pipeline running as %q (preemptible VM:%t)\n", lro.Name, req.Pipeline.Resources.VirtualMachine.Preemptible)
```

Comment:
I think if you want to put the preemptibility in this message we should:
(1) include the attempt number
(2) drop the attempt number from the message below

So we'd have

"Pipeline running as %q (attempt %d, preemptible: %t)", attempt+1, req.Pipeline.Resources.VirtualMachine.Preemptible

and just

"Execution failed: %v", err